Extension not found in catalog

Trying to use quarkus CLI to install dapr extension I am getting the following error:

```bash
custom-app quarkus ext add io.quarkiverse.dapr:quarkus-dapr
Looking for the newly published extensions in registry.quarkus.io
[ERROR] ❗  Nothing installed because keyword(s) 'io.quarkiverse.dapr:quarkus-dapr' were not matched in the catalog.
```

--- a/.github/project.yml
+++ b/.github/project.yml
@@ -1,4 +1,4 @@
 release:
-  current-version: 1.0.4
-  next-version: 1.0.5-SNAPSHOT
+  current-version: 2.0.0
+  next-version: 999-SNAPSHOT
 

